tests/contrib/fastapi/test_fastapi.py

```diff
@@ -606,13 +620,15 @@ def test_table_query_snapshot(snapshot_client):
     }
 
 
-def test_background_task(client, tracer, test_spans):
-    """Tests if background tasks have been excluded from span duration"""
-    response = client.get("/asynctask")
+@snapshot(ignores=["meta._dd.span_links"])
```

Comment:
Ignoring the span link attributes for now, because these are not [normalized like the span ID and trace IDs in traces](https://github.com/DataDog/dd-apm-test-agent/blob/97d78ff8478737f164982b58b867737b33fbc69b/ddapm_test_agent/trace_snapshot.py#L86) by the test agent. Issue is tracked [here](https://github.com/DataDog/dd-apm-test-agent/issues/154)